damnit/backend/listener.py

```diff
@@ -110,7 +111,11 @@ def handle_event(self, record, msg: dict, run_data: RunData):
         log.info(f"Added p%d r%d ({run_data.value} data) to database", proposal, run)
 
         req = ExtractionRequest(run, proposal, run_data)
-        self.submitter.submit(req)
+        try:
+            self.submitter.submit(req)
+        except Exception:
+            log.warning("Slurm job submission failed, starting process locally")
```

Comment:
Might be worth logging some info about the exception here, e.g. `exc_info=True` to include a traceback in the logs. We could suppress that for `FileNotFoundError`, which is what people should see without Slurm available.

Reply:
good point, thanks